### PR TITLE
update sbt and sbt plugins

### PR DESCRIPTION
**Pull request by He-Pin**

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.2.6
+sbt.version=1.2.7
```
Comment (helena) [inline project/build.properties]: 👍

**Comment by helena** [comment]
My only concern with the plugins upgrades is - does CI test all tasks relevant for confidence in the plugins upgrades?

**Review by raboof — APPROVED** [review]
LGTM, thanks! @helena CI does not touch everything that will be touched when a release is done, but since it's all minor versions it seems reasonable to assume it'll be OK and fix any regressions when we actually release.

**Review by chbatey — APPROVED** [review]
LGTM, thanks!

**Review by helena — APPROVED** [review]
LGTM!

**Pull request merged**